README.md

```diff
@@ -1,9 +1,11 @@
 ## Make it So
 
-This is the source code for Make It So, a sample app that demonstrates how to use Firebase in an Android app. The first version of this app was built in 2022 and you can find it in the v1 folder - there you will also find instructions to set up your Firebase project and links to articles, documentation and codelabs that guide you through the code. 
+This is the source code for Make It So, a sample app that demonstrates how to use Firebase in an Android app. 
 
-We're currently working on v2.
+The first version of this app was built in 2022 and you can find it in the v1 folder - there you will also find instructions to set up your Firebase project and links to articles, documentation and codelabs that guide you through the code. 
+
+Version 2 of this app was built in 2025 and you can find it in the v2 folder - there you will also find instructions to set up your Firebase project and connect it to the Make it So app.
```

Comment:
Should we also explain what is the difference between v1 and v2?

Reply:
Added!